app/src/main/java/com/zendesk/rememberthedate/model/DateModel.java

```diff
@@ -2,21 +2,31 @@
 
 import android.text.format.Time;
 
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
 public class DateModel {
 
-    private String title;
-    private Time time;
+    private final long id;
+    private final String title;
+    private final Date date;
```

Comment:
Bit of a mad question here - did we consider storing a `long` of the Date's milliseconds/timestamp rather than the `Date` object itself? `getDate()` could've returned `new Date(timestampInMillisOrWhatever)` instead and the constructor could've taken a `long` instead of a `Date`. It's not a big issue, just curious if it was considered or if that approach had a problem I haven't thought of.